instance/static/scss/instance.scss

```diff
@@ -23,6 +39,23 @@ ul.side-nav li.active {
         max-width: none;
     }
 
+    .scrollable {
+        overflow: auto;
+        height: calc(100vh - #{$header-height});
+    }
+
+    .scrollable::-webkit-scrollbar {
```

Comment:
@UmanShahzad I assume you know what you're doing by using vendor-specific pseudo-elements, especially those that don't have analogs with other popular browser vendors (i.e. firefox and ie):

https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar

Does it make sense to completely remove them or replicate the same effect using "some creativity", like [this SO answer](https://stackoverflow.com/a/30464993/882918) suggests?

Reply:
@UmanShahzad I'd like to point out it also affected other scrollables, at least for me - note right scrollbar is also styled the same way:

![image](https://user-images.githubusercontent.com/3330048/35044122-43b76268-fba0-11e7-82e1-4e87f33f6fb0.png)

While on prod it uses native scrollbar:

![image](https://user-images.githubusercontent.com/3330048/35044145-584bdfb0-fba0-11e7-931f-9c799bed40b1.png)

Unless it is intentional there might be a problem of (re)using too generic css class.

Reply:
Yeah I knew about this being vendor-specific, as you can [see in my comment above](https://github.com/open-craft/opencraft/pull/267#issuecomment-357342018). But it's just styling so I didn't bother with alternatives. I saw that SO answer but if you see the actual CSS suggested, that is very ugly, fragile code. If you insist, I can just delete this styling, but I thought it's better to have for at least some, than not having it at all.

Reply:
Yes the 2nd scrollbar is intentional: that div itself becomes scrollable. See what happens when you make it responsive -- the actual browser scrollbar is not affected.

Reply:
@UmanShahzad That's the question of taste of curse, so do what you see fit best, but is there particular reason you wanted a custom scrollbar in the first place (as opposed to just let browser use native one?)

Reply:
No good reason, I just wanted to spice it up ;)

Reply:
@UmanShahzad I'd slightly prefer going with native control just to reduce the amount of code (and non-standard solutions), but since I'm not the author of this PR I'll leave it up to your good judgement.

Reply:
Good point, but since I already invested the effort to get that to work, and it doesn't negatively effect any users of non-Chrome browsers, I will opt to leave this here. Oh and then I can hit merge without waiting again :D